client/src/components/Libraries/LibraryFolder/TopToolbar/FolderTopBar.vue

```diff
@@ -352,9 +429,25 @@ onMounted(async () => {
             </div>
         </div>
 
+        <BAlert v-model="progress" :dismissible="progressStatus.runningCount === 0" variant="info" class="mb-1">
+            <ProgressBar
+                :loading="progressStatus.runningCount > 0"
+                :note="progressNote"
+                :total="progressStatus.total"
+                :ok-count="progressStatus.okCount"
+                :error-count="progressStatus.errorCount"
+                :running-count="progressStatus.runningCount" />
+        </BAlert>
+
         <LibraryBreadcrumb
             v-if="props.metadata && props.metadata.full_path"
             :full_path="props.metadata.full_path"
             :current-id="props.folderId" />
+
+        <HistoryDatasetPicker
+            v-if="modalShow === 'history'"
+            :folder-id="props.folderId"
```

Comment:
Where in the HistoryDatasetPicker is this folder id used?

Reply:
I also wonder if the `HistoryDatasetPicker` should replace the `DataDialog`, it seems that we have two dataset dialogs now, one with pagination and one without.

Reply:
> Where in the HistoryDatasetPicker is this folder id used?

It looks like the `folderId` isn’t used in `HistoryDatasetPicker`. It seems like it was left over and can be safely removed.

Reply:
> DataDialog

`DataDialog` and `HistoryDatasetPicker` both utilize the `SelectionDialog`, though each in a different way.